src/__tests__/App.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '@testing-library/react';
+import renderer from 'react-test-renderer';
+import App from '../App';
+
+describe('App', () => {
+  it('renders', () => {
+    const tree = renderer
+      .create(
+        <App />,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Well done for writing detailed tests. We are only left with one thing to do. That is the simulation of user interactions with React Testing Library as required of us in the project requirements. That requirement is quoted below:
  
  > - Use React Testing Library to simulate user interaction.
  
  ## Suggestion
  One user interaction we can implement is clicking a combination of keys and lastly the (=) key using the `fireEvent` method from `@testing-library/react` and then asserting that the result of that calculation is present on the page using the `getByText` method.
  
  This simple [GUIDE](https://www.pluralsight.com/guides/simulate-browser-events-react-react-testing-library) from Pluralsight can really help with this.